lib/praxis-mapper/identity_map.rb

```diff
@@ -119,7 +120,7 @@ def load(model, &block)
       end
 
       records = query.execute
-      add_records(records)
+      actually_added = add_records(records)
```

Comment:
maybe it's silly, but since add_records always returns all records (in the IM), calling it `actually_added` seems a little misleading? how about something that has the feel of "the corresponding records in the IM"...like `backing_records` or `im_records` ...?